src/Hit/Issue.hs

```diff
@@ -164,3 +168,18 @@ parseOwnerRepo url =
 
     stripGitSuffix :: Text -> Maybe Text
     stripGitSuffix x = whenNothing (T.stripSuffix ".git" x) (Just x)
+
+fetchIssueStrict :: Id Issue -> IO Issue
+fetchIssueStrict iNum = fetchIssue iNum >>= \case
+    Left err -> errorMessage (show err) >> exitFailure
+    Right issue -> pure issue
```

Comment:
It starts to look like we have reimplemented the same code many times. Now we have the following functions:

```haskell
fetchIssue :: Id Issue -> IO (Either Error Issue)
fetchIssue iNum = withOwnerRepo (\t o r -> issue' t o r iNum)

getIssue :: Id Issue -> IO ()
getIssue num = fetchIssue num >>= \case
    Left err -> errorMessage $ show err
    Right is -> putTextLn $ showIssueFull is

getIssueTitle :: Id Issue -> IO Text
getIssueTitle num = fetchIssue num >>= \case
    Left err -> errorMessage (show err) >> exitFailure
    Right Issue{..} -> pure issueTitle
```

And now `fetchIssueStrict`... All these functions are useful. but I guess it's better to remove code duplication and reuse what we've already written.

1. Every time where `fetchIssue` is used, we show errors and exit with failure or just return the issue. So I propose to change the type and the implementation of `fetchIssue` to do exactly that.
2. `getIssue` should be renamed to `printFullIssue`
3. `getIssueTitle` should use `Functor` to extract title from the result of `fetchIssue`.
4. And now `fetchIssueStrict` is no longer needed.